src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration.mdx

```diff
@@ -20,56 +20,53 @@ redirects:
+    <figcaption>
+    Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Kubernetes**</DoNotTranslate> and choose your cluster. The cluster explorer is our handy, all-in-one solution for all your Kubernetes needs when you're running at large scale.
+    </figcaption>
+    
 
-You'll be able to configure Kubernetes to suit your environment after you create a New Relic account (it's free, forever!) and [install our Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/).
+## Monitoring Kubernetes with New Relic [#monitoring-k8s-nr]
 
-<InlineSignup/>
+Managing Kubernetes can be tough. There's a lot going on all the time, with containers being created and deleted in minutes, apps crashing, and resources being used in unexpected ways. Our integration helps you figure out how to use Kubernetes in different ways, whether you're using it on-premises, in the cloud, or in a hybrid setup.
 
-## Why it matters [#features]
+Take a look at what you can do with New Relic:
 
-Governing the complexity of Kubernetes can be challenging; there's so much going on at any given moment, with containers being created and deleted in a matter of minutes, applications crashing, and resources being consumed unexpectedly. Our integration helps you navigate Kubernetes abstractions across on-premises, cloud, and hybrid deployments.
+* You can monitor the [Kubernetes control plane](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring) to manage and collect metrics from your clusters.
 
-In New Relic, you can build your own charts and [query](/docs/using-new-relic/data/understand-data/query-new-relic-data) all your [Kubernetes data](/docs/integrations/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data), which our integration collects by instrumenting the container orchestration layer. This gives you additional insight into nodes, namespaces, deployments, replica sets, pods, and containers.
+* You can [forward your Kubernetes logs](/docs/logs/forward-logs/kubernetes-plugin-log-forwarding/) to New Relic, which will give you more ways to manage your log data. You can collect, process, explore, query, and alert on your log data.
```

Comment:
Can you rewrite these to eliminate the "You can" and instead use an action verb that provides a value proposition?  Like, "Collect, process, explore, query, and alert on your log day through Kubernetes logs"